lib/vagrant-service-manager/command.rb

```diff
@@ -183,7 +183,7 @@ def restart_service(service)
                     # to OpenShift service
                     "sccli #{service}"
```

Comment:
This could be a problem.  People may expect each operation to do somethign specific and with an sccli service they do not.   This goes to what @hferentschik was saying about needing a way to separate "generic services" from "core services" (my words).